criteria/require-review.md

```diff
@@ -6,6 +6,8 @@ order: 7
 <!-- SPDX-License-Identifier: CC0-1.0 -->
 <!-- written in 2022 by The Foundation for Public Code <info@publiccode.net> -->
 
+This is a short guide that can be useful if you haven't thought about reviewing code from external contributors before: [10 universal steps for open source code review](https://opensource.com/article/22/10/code-review), by Red Hat.
+
```

Comment:
This would fit better in "Further reading" rather than above requirements.

Reply:
Sorry, my head was in the wrong repo. This is fine.

Reply:
Ah yes, it could work as a H2 in the end.

Reply:
Could, but I think it is fine as-is.